cfgov/v1/atomic_elements/molecules.py

```diff
@@ -257,6 +257,7 @@ class ContactPhone(blocks.StructBlock):
     phones = blocks.ListBlock(
         blocks.StructBlock([
             ('number', blocks.CharBlock(max_length=15)),
+            ('extension', blocks.CharBlock(max_length=4, required=False)),
             ('vanity', blocks.CharBlock(max_length=15, required=False)),
             ('tty', blocks.CharBlock(max_length=15, required=False)),
```

Comment:
And also put in an explicit label on this field so that it capitalizes the whole thing.